main.js

```diff
@@ -16,17 +36,15 @@ function displayNumbers(randomNumbers, currentScore) {
     const numbersOnScreen = randomNumbers.length;
     let selectedOrder = [];
 
-    document.querySelector('#play_area').textContent = '';
+    document.querySelector('#play_area').innerHTML = grid;
 
     // using a for loop instead of forEach as the array.pop changes the length over each iteration
     for (let i = 0; i < numbersOnScreen; i++) {
-        let randNumber = randomNumbers.pop();
-        let number = document.createElement('div');
-
-        number.textContent = randNumber;
-        number.setAttribute('class', 'numberBox');
+        const cards = document.querySelectorAll('.placeholder:not(.numberBox)');
```

Comment:
any reason the :not needs to be here? didn't know about this tho, cool

Reply:
This stops any numbers already populated in the grid being over-written.